src/Bot/Log.hs

```diff
@@ -49,6 +55,20 @@ recordUserMsg sender msg = do
         }
   return ()
 
+getRecentLogs :: Seconds -> Effect [LogRecord]
+getRecentLogs offset = do
+  currentTime <- now
+  let diff = intToNominalDiffTime $ -1 * offset
```

Comment:
We have a small problem here that crashes the whole bot:

```haskell
λ> :m + Data.Time
λ> (fromInteger $ toInteger (-2 :: Seconds)) :: NominalDiffTime 
*** Exception: arithmetic underflow
```

Please move the minus outside:

```haskell
λ> -(fromInteger $ toInteger (2 :: Seconds)) :: NominalDiffTime 
-2s
```

Reply:
it was a silly one(
fixed and implemented unit test.